fitbenchmarking/fitbenchmarking
# Update README

#### Description of Work

Fixes [#727](https://github.com/fitbenchmarking/fitbenchmarking/issues/727)

#### Testing Instructions

1. Do links work as expected?

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
